generate/resources/couchbase-edge-server/README.md

```diff
@@ -1,16 +1,101 @@
+
+**Step - 2 :** Start the docker container in interactive mode with the previously created `etc` directory mounted.  This is accomplished by setting the entry point of the container to `bash`. 
+
 ```
+docker run -it --rm --entrypoint=bash -v $LOCALPATH/etc:/opt/couchbase-edge-server/etc couchbase/edge-server
+```
+
+**Step - 3 :** Create a TLS key and certificate for use with the server.  Normally, this would be done through your provider and the instructions are out of scope for this README, but for testing and tutorial purposes the edge server can create some self signed certificates for you.  The following command will create a cert and key file in the mounted `etc` directory (the default working directory of the container)
+
+```
+couchbase-edge-server --create-cert 127.0.0.1 cert.pem key.pem
+```
+
+**Step - 4 :** While still in the container, create a user for the server so that HTTP auth can be used.  This will create a user named alice..
+
+```
+echo "{}" > users.json
+couchbase-edge-server --add-user users.json alice
+```
+
+This will prompt you twice for a password.  Pick whatever you like, I will refer to it as `$PASSWORD`.
+
+**Step - 5 :** After exiting the container, now it is time to write a config.json into the `$LOCALPATH/etc` folder that will use the items that were just created.  Create a config.json file with the following content:
+
+```json
+{
+    "$schema": "https://packages.couchbase.com/couchbase-edge-server/config_schema.json",
+    "https": {
+        "tls_cert_path": "/opt/couchbase-edge-server/etc/cert.pem",
+        "tls_key_path": "/opt/couchbase-edge-server/etc/key.pem"
+    },
+    "users": "/opt/couchbase-edge-server/etc/users.json",
+    "databases": {
+        "db": {
+            "path":  "/opt/couchbase-edge-server/var/databases/example.cblite2",
+            "create": true,                   
+            "enable_client_writes": true,
+            "enable_client_sync": true
+        }
+    }
+}
+```
+
+**Step - 6 :** Now all that's left to do is to start the container in server mode with both the `etc` and `databases` directories mounted.
+
+```
+docker run -d -p 59840:59840 -v $LOCALPATH/etc:/opt/couchbase-edge-server/etc -v $LOCALPATH/databases:/opt/couchbase-edge-server/var/databases couchbase/edge-server
+```
+
+Once started, you will notice that your `$LOCALPATH/databases` folder now has a database called `example.cblite2` inside of it, due to the `create: true` portion of the config.  Accordingly, if you have pre-existing databases that you want to use, you can include them in this folder and add the appropriate section to the `databases` section of the config.
+
+You'll also notice that now in order to access the REST API you will need to use HTTP basic auth as user `alice` with password `$PASSWORD`.  If you were to use a real TLS certificate instead, along with users applicable to your use case, then this would be a production deployment.
```

Comment:
Add that, after making any changes to the config or users file, you need to restart the server.